extensions/2.0/Khronos/KHR_spz_compression/README.md

```diff
@@ -0,0 +1,208 @@
+# KHR\_spz\_compression
+
+## Contributors
+
+- Jason Sobotka, Cesium
+- Adam Morris, Cesium
+- Sean Lilley, Cesium
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Adding Gaussian Splats](#adding-spz-compressed-data-to-primitives)
+  - [Extension Properties](#extension-properties)
+  - [Accessors](#accessors)
+- [Conformance](#conformance)
+- [Implementation](#implementation)
+- [Schema](#schema)
+- [Known Implementations](#known-implementations)
+- [Resources](#resources)
+
+## Overview
+
+SPZ is a compression format from Niantic Labs designed for Gaussian splats. Currently, it is open sourced under the MIT license. The SPZ format is primarily used in the Niantic Labs Scaniverse app. It was purpose built for Guassian splats and offers a balance of high compression with minimal visual fidelity loss.

```

Comment:
```suggestion
SPZ is a compression format from Niantic Labs designed for Gaussian Splatting and available as open source under the MIT licensed. SPZ was purpose built for Gaussian Splatting and offers a balance of high compression with minimal visual fidelity loss.
```

We can't mention Scaniverse, and we should avoid terms like "Currently" in any specifications.